src/Components/Nav/Nav.js

```diff
@@ -0,0 +1,183 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Category from './Components/Category';
+
+import './Nav.scss';
+
+class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isCategoryShown: false,
+      navBarFixed: false,
+    };
+  }
+
+  handleCategoryOn = () => {
+    this.setState({
+      isCategoryShown: true,
+    });
+  };
+
+  handleCategoryOff = () => {
+    this.setState({
+      isCategoryShown: false,
+    });
+  };
+
+  componentDidMount() {
+    window.addEventListener('scroll', this.scrollNavBar);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.scrollNavBar);
+  }
+
+  scrollNavBar = () => {
+    const scroll = window.scrollY;
+    const { navBarFixed } = this.state;
+
+    const scrollDown = scroll > 150 && this.state.navBarFixed === false;
+    const scrollUp = scroll <= 150 && this.state.navBarFixed === true;
```

Comment:
```suggestion
    const scrollDown = scroll > 150 && !this.state.navBarFixed;
    const scrollUp = scroll <= 150 && this.state.navBarFixed;
```
truthy한 값을 그대로 사용해줄 수 있습니다

Reply:
감사합니다